proto/ptypes/replica.proto

```diff
@@ -42,6 +42,11 @@ message ReplicaExpandRequest {
     int64 size = 1;
 }
 
+message ReplicaExpandReply {
+    Replica replica = 1;
+    bool rollback_failed = 2;
```

Comment:
if `rollback_failed == false`, how did we know if the volume expansion is successful or not?

Reply:
or
```
bool expansion_result = 2;
string expansion_error_msg = 3;
```
So if expansion failed but roll back succeed, the function return error nil and set expansion_result = false and error message.
If the expansion failed and rollback failed, the function return error fo the rollback, and set the expansion result = false and error message for expansion as well.